src/librustc_middle/mir/interpret/error.rs

```diff
@@ -422,6 +427,11 @@ impl fmt::Debug for UndefinedBehaviorInfo {
                 "using uninitialized data, but this operation requires initialized memory"
             ),
             DeadLocal => write!(f, "accessing a dead local variable"),
+            ArgumentSizeMismatch { target_size, data_size } => write!(
```

Comment:
This function is not just used for arguments, it is used for scalars occurring anywhere in the program. Thus, I think `ScalarSizeMismatch` is a better name.

Reply:
:+1: Changed this